src/engine/QMLEngine.js

```diff
@@ -141,6 +141,10 @@ class QMLEngine {
     return basePath.join("/");
   }
 
+  extractFileName(file) {
+    return file.split('\\').pop().split('/').pop();
```

Comment:
Not sure why, but that does not seem to work for me. Basically makes it try to load "./qml/.,qml,index.qml"